README.md

```diff
@@ -1,46 +1,37 @@
-[![npm version](https://badge.fury.io/js/postcss-increase-specificity.svg)](http://badge.fury.io/js/postcss-increase-specificity) [![Build Status](https://travis-ci.org/MadLittleMods/postcss-increase-specificity.svg)](https://travis-ci.org/MadLittleMods/postcss-increase-specificity)
 
 # PostCSS Increase Specificity
 
-[PostCSS](https://github.com/postcss/postcss) plugin to increase the specificity of your selectors.
+[PostCSS](https://github.com/postcss/postcss) plugin to increase the specificity of our selectors.
```

Comment:
@sagyos Can you provide some context behind these changes? What is it doing? Why? What use cases?